### PR TITLE
Style all tables

### PR DESCRIPTION
**Pull request by stephanebisson**
Phabricator Link: https://phabricator.wikimedia.org/T248716

### Problem Statement

Some tables are completely hidden.

### Solution

Style all tables but hide the infobox explicitly.

### CODE REVIEW

**Comment by stephanebisson** [comment]
Need to fix the tests

**Comment by hueitan** [comment]
looks good to me! Thanks for making it general!

**Comment by stephanebisson** [comment]
@jpita can you help with troubleshooting the tests? I don't understand what's going on.

**Comment by jpita** [comment]
@stephanebisson in the cat article, when you open the quick facts, nothing is selected/with focus.
I believe that on master the first image is on focus and then we go to the first link navigating with the arrows, which is different on this branch.

**Comment by stephanebisson** [comment]
> @stephanebisson in the cat article, when you open the quick facts, nothing is selected/with focus.
> I believe that on master the first image is on focus and then we go to the first link navigating with the arrows, which is different on this branch.
> 

Good catch! That's exactly it.

File: style/variables.less [hunk 1]
```diff
@@ -13,7 +13,9 @@
 @smallFont: 12px;
 @bigFont: 17px;
 
-@data-selected-selector: &[ data-selected='true'];
+@data-selected-selector: &[data-selected='true'];
```
Comment (medied) [inline style/variables.less:16]: A step towards towards [T242364](https://phabricator.wikimedia.org/T242364) 👍

**Pull request merged**